code/__DEFINES/__game.dm

```diff
@@ -208,3 +208,6 @@ var/static/list/scarySounds = list('sound/weapons/thudswoosh.ogg','sound/weapons
 
 // Default preferences
 #define DEFAULT_SPECIES "Human"
+
+//Husking Switch For Burning
+#define HUSKING_ON 0
```

Comment:
Make it configuration var on game_options.dm rather than a define, so it is actually toggleable.
Also `FALSE`.